src/Suave/Owin.fs

```diff
@@ -634,8 +634,17 @@ module OwinApp =
         HttpContext.request_ >--> HttpRequest.headers_,
         HttpContext.response_ >--> HttpResult.headers_
 
+      let handleResponse  (rhs : Dictionary<string, string[]>) = 
+        let handleKey key =
```

Comment:
Hmm, I guess you could argue it's a Header const.  Any preference to where Headers.fs/Cookies.fs?